branding/css/susemanager-theme.less

```diff
@@ -640,6 +640,7 @@ header, nav.navbar-pf {
   display: inline-block;
   width: 5em;
   border: 1px solid #dddddd;
+  background-color: #ebebeb;
```

Comment:
I would like to add a global variable to be reused for this color. Maybe you can add it [here](https://github.com/uyuni-project/uyuni/blob/master/branding/css/susemanager-variables.less#L11) something like `@spacewalk-gray-light-background: #ebebeb` and reuse it here with `background-color: @spacewalk-gray-light-background;`. With this we can reuse the same background if this fix needs to be applied somewhere else.

Reply:
fixed